File: Weatley.Backend/Weatley.Model/Entities/Activity.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Weatley.Model.Entities
{
    public class Activity : IEntityBase
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public DateTime StartHour { get; set; }
        public DateTime EndHour { get; set; }
    
    }
}

File: Weatley.Web/src/app/core/entities/activity.ts
import { Hotel } from './hotel';

export class Activity {
	id: string;
	name: string;
	description: string;
	startHour: Date;
	endHour: Date;
	hotel: Hotel;

	constructor(values: Object = {}) {
		Object.assign(this, values);
	}
}
